tasks/bootstrap_local_vm/05_add_host.yml

```diff
@@ -59,6 +59,7 @@
       state: present
       name: "{{ he_cluster }}"
       data_center: "{{ he_data_center }}"
+      cpu_type:  "{{ he_cluster_cpu_type | default(omit) }}"
```

Comment:
I think it will be better to add he_cluster_cpu_type variable to README file in https://github.com/oVirt/ovirt-ansible-hosted-engine-setup/blob/master/README.md#general-variables